### PR TITLE
QA Updated KubeVirt references

### PR DESCRIPTION
**Pull request by qahmed1998**

### CODE REVIEW

**Comment by qahmed1998** [comment]
@miq-bot assign @Fryguy

**Comment by qahmed1998** [comment]
@miq-bot add_labels radjabov/yes?, quinteros/yes?

File: _includes/provider-ocp-add-container.md [hunk 1]
```diff
@@ -109,7 +109,7 @@
 
 9. In the **Virtualization** endpoint tab optionally configure the virtualization service details:
 
-    1. Select a service type (KubeVirt) or leave disabled
+    1. Select a service type (OpenShift Virtualization (KubeVirt)) or leave disabled
```
Comment (Fryguy) [inline _includes/provider-ocp-add-container.md:112]: @agrare This requires a code change.

@qasim-ahmed Can we change this one to say `OpenShift Virtualization / KubeVirt`?  That's what the UI value will be when we make the code change, so I want it to match here.

File: capabilities_matrix/_topics/infrastructure_providers.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
  The following tables outline the capabilities of {{ site.data.product.title_short }} that are known to work for various virtual infrastructure providers.
 
-| Discovery                                                    | vSphere | oVirt / RHV | OpenStack undercloud    | IBM Power HMC | KubeVirt |
+| Discovery                                                    | vSphere | oVirt / RHV | OpenStack undercloud    | IBM Power HMC | OpenShift Virtualization (KubeVirt) |
```
Comment (Fryguy) [inline capabilities_matrix/_topics/infrastructure_providers.md:5]: Actually, now that I made that other comment, it should be `KubeVirt / OSV` in these docs to match the pattern of `oVirt / RHV`  (the pattern being "opensource name" / "Red Hat productized name")

File: capabilities_matrix/_topics/providers.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ The following table lists providers that {{ site.data.product.title_short }} can
 | [IBM PowerVC](#cloud-providers)                                                      |
 | IBM Terraform                                                                        |
 | [Kubernetes](#container-providers)                                                   |
-| [KubeVirt](#infrastructure-providers)                                                |
+| [OpenShift Virtualization (KubeVirt)](#infrastructure-providers)                     |
```
Comment (Fryguy) [inline capabilities_matrix/_topics/providers.md:20]: In this case, please make these 2 separate line items.  That's what we've done for oVirt and Red Hat Virtualization.  Additionally, this particular screen should spell it out so, for OSV use the phrase `Red Hat OpenShift Virtualization (OSV)`, and then `KubeVirt` separately. (and of course in alphabetical order).

**Comment by Fryguy** [comment]
Everything else looks good to me. :+1:

File: _includes/provider-ocp-add-container.md (hunk 1 above)
Comment (Fryguy) [reply _includes/provider-ocp-add-container.md:112]: https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/532

**Review by Fryguy — APPROVED** [review]
I pushed a small commit to fix a couple minor things, so this is ready to merge.

**Comment by Fryguy** [comment]
Backported to `radjabov` in commit f4ca9413571b37150040ab588e806831b2a87342.

```text
commit f4ca9413571b37150040ab588e806831b2a87342
Author: Jason Frey <fryguy9@gmail.com>
Date:   Wed Jul 17 15:52:35 2024 -0400

    Merge pull request #1797 from qahmed1998/QA-update-os-virtualization
    
    QA Updated KubeVirt references
    
    (cherry picked from commit c92dcf0573a1408e7020065f8f54043682982729)
```

**Comment by Fryguy** [comment]
Backported to `quinteros` in commit 94af716ab66c9b811416804b389aa407c0e0da34.

```text
commit 94af716ab66c9b811416804b389aa407c0e0da34
Author: Jason Frey <fryguy9@gmail.com>
Date:   Wed Jul 17 15:52:35 2024 -0400

    Merge pull request #1797 from qahmed1998/QA-update-os-virtualization
    
    QA Updated KubeVirt references
    
    (cherry picked from commit c92dcf0573a1408e7020065f8f54043682982729)
```

**Pull request merged**